src/tests/unit/tests/common/components/cards/card-kebab-menu-button.test.tsx

```diff
@@ -144,6 +144,22 @@ describe('CardKebabMenuButtonTest', () => {
         expect(rendered.find(ActionButton).prop('menuProps')).toMatchSnapshot();
     });
 
+    it.each(['test-kebabmenuarialabel', undefined])(
+        'renders per snapshot with allCardInteractionsSupported and %s aria label passed as prop',
+        ariaLabel => {
+            const newProps = {
+                ...defaultProps,
+                kebabMenuAriaLabel: ariaLabel,
+            };
+            const rendered = shallow(
+                <CardKebabMenuButton {...newProps} deps={{ ...defaultDeps, cardInteractionSupport: allCardInteractionsSupported }} />,
+            );
+
+            expect(rendered.debug()).toMatchSnapshot();
+            expect(rendered.find(ActionButton).prop('menuProps')).toMatchSnapshot();
```

Comment:
I think in general, when having more than one `.toMatchSnapshot();` in the same test, we add a comment so it's a little easier to differentiate the snapsnots on the .snap file

Reply:
if you look at the .snap file on this PR it shows:
`CardKebabMenuButtonTest renders per snapshot with allCardInteractionsSupported and undefined aria label passed as prop` witth 2 different suffixes: `1` and `2`... not sure which one is which, having the comment helps readability

Reply:
Sure. Will add a comment.